packages/web-twig/src/Resources/components/TextFieldBase/TextFieldBase.twig

```diff
@@ -68,9 +68,7 @@
 {% endif %}
 
 <div {{ mainProps(_mainPropsWithoutReservedAttributes) }} {{ styleProp(_styleProps) }} {{ classProp(_rootClassNames) }} {{ _dataToggleAttr }}>
-    <label for="{{ _id }}" {{ classProp(_labelClassNames) }}>
-        {% if _unsafeLabel %}{{ _unsafeLabel | raw }}{% else %}{{ _label }}{% endif %}
-    </label>
+    <label for="{{ _id }}" {{ classProp(_labelClassNames) }}>{% if _unsafeLabel %}{{ _unsafeLabel | raw }}{% else %}{{ _label }}{% endif %}</label>
```

Comment:
I'd prefer using the right tool for the job as we don't have to sacrifice readability of the code 🙂.

```suggestion
    <label for="{{ _id }}" {{ classProp(_labelClassNames) }}>
      {%- if _unsafeLabel %}{{ _unsafeLabel | raw }}{% else %}{{ _label }}{% endif -%}
    </label>
```

or even more readable:

```suggestion
    <label for="{{ _id }}" {{ classProp(_labelClassNames) }}>
      {%- if _unsafeLabel -%}
        {{ _unsafeLabel | raw }}
      {%- else -%}
        {{ _label }}
      {%- endif -%}
    </label>
```

Reply:
(I'm aware the other form field components rather use spaghetti code. One day, I'd like to improve their coding style too.)

Reply:
Thanks for a suggestion. Much better way how to prevent unwanted blank space. :) Don't know, why it doesn't came to my mind before.